packages/shared/src/components/Logo.tsx

```diff
@@ -103,7 +105,7 @@ export default function Logo({
       <a
         aria-disabled={linkDisabled}
         className={classNames(
-          'relative flex items-center',
+          'flex items-center',
```

Comment:
Doesn't the plus badge need this relative position? (not 100% sure)

Reply:
I movted it to initial which was empty and it should cover all cases.
So basically absolute option defines `absolute`, and relative defines `relative`, the only option not covered is initial